catalogue/terraform/stack/main.tf

```diff
@@ -42,6 +46,17 @@ module "path_listener" {
   priority      = "49997"
 }
 
+module "api_path_listener" {
```

Comment:
N.B. tried using [next.js rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites), so this wouldn't be necessary but it doesn't work with the api routes